api/controllers/mentor.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
         include: [
           {
             model: models.Users,
+            attributes: {},
```

Comment:
is there a reason you passing attributes as an empty object????

Reply:
File still in development, Kindly ignore